### DISCUSSION TITLE
Why Doesn't container.make Work in an AdonisJS 6 Command Class?

### DISCUSSION THREAD

**Discussion by atakansn** [adonisjs/core#4776] [Help] [answered]
In package development(pkg-starter-kit) in AdonisJS 6, why doesn't container.make perform the call inside a command class?

```
export default class MakeFoo extends BaseCommand {
    async run() {
      const app = await this.app.container.make('app');
    }
 }
```

RuntimeException: Cannot resolve binding "app" from the container

**Comment by RomainLanz** [comment, marked as answer]
Hey @atakansn! 👋🏻 

Have you started the application by setting the option `startApp` to `true`?

📚 https://docs.adonisjs.com/guides/ace/creating-commands#command-metadata

**Reply by atakansn** [reply]
Thanks!
